bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -205,38 +202,29 @@ found_file = os.path.abspath(opts.found_file) if opts.found_file else None
 zoom_in = opts.zoom_in
 veto_type = opts.y_variable
 ifo = opts.ifo
-snr_type = 'snr'
+snr_type = opts.snr_type
 # If this is false, coherent SNR is used on the horizontal axis
 # otherwise the single IFO SNR is used
-use_sngl_ifo_snr = opts.use_sngl_ifo_snr
-if use_sngl_ifo_snr:
+if snr_type == 'single':
     if ifo is None:
         err_msg = "--ifo must be given when using --use-sngl-ifo-snr"
         parser.error(err_msg)
-    snr_type += '_sngl'
-# If this is false, network vetoes are used on the vertical axis
-# otherwise the veto is intended as a single IFO quantity
-use_sngl_ifo_veto = opts.use_sngl_ifo_veto
-if use_sngl_ifo_veto:
-    if ifo is None:
-        err_msg = "--ifo must be given when using --use-sngl-ifo-snr-veto"
-        parser.error(err_msg)
-    elif veto_type != 'standard':
-        err_msg = "Single IFO values are available only for the standard chi-square"
-        parser.error(err_msg)
-    else:
-        veto_type += '_sngl'
```

Comment:
All this should be kept when rescuing the ability to look at single IFO veto values.

Reply:
Please track `use_sngl_ifo_veto` in the remainder of the code, and reinsert it.